consensus/enclave/trusted/Cargo.toml

```diff
@@ -39,7 +39,6 @@ mbedtls = { version = "0.8.1", default-features = false, features = ["no_std_dep
 mbedtls-sys-auto = { version = "2.26.1", default-features = false, features = ["custom_threading"] }
 
 # We include these here so we can select in-the-enclave features
-cpufeatures = { version = "0.1.5", features = ["compile-only"] }
```

Comment:
So we're changing the patch version and removing it?

Reply:
cpufeatures is used directly by the RustCrypto crates (aes-gcm, sha2, maybe blake2, etc.).

This line was there to ensure the "compile-only" feature was turned on, via feature unification, but not actually used in the trusted crate.

Since feature unification doesn't work that way, there's no need for this line to exist.

We still patch in our fork in the enclave so our fork is the one used by the upstream RustCrypto crates.